meshroom/core/__init__.py

```diff
@@ -255,7 +255,9 @@ def loadSubmitters(folder, packageName):
 subs = loadSubmitters(meshroomFolder, 'submitters')
 # -  additional 3rd party submitters
 if "MESHROOM_SUBMITTERS_PATH" in os.environ:
-    subs += loadSubmitters(os.environ["MESHROOM_SUBMITTERS_PATH"], 'submitters')
+    subs = loadSubmitters(os.environ["MESHROOM_SUBMITTERS_PATH"], 'submitters')
+else:
+    subs = loadSubmitters(meshroomFolder, 'submitters')
```

Comment:
In that case, it could be:
`subs = loadSubmitters(os.environ.get("MESHROOM_SUBMITTERS_PATH", meshroomFolder), 'submitters')`

Reply:
Agreed, that is a bit nicer.  Shall I amend the PR?